cypress/integration/computation_modern_combinations_pt1.spec.ts

```diff
@@ -0,0 +1,71 @@
+/// <reference types="cypress" />
+
+describe("ComputationMenu", () => {
+    beforeEach(() => {
+        cy.clearLocalStorage();
+        cy.viewport(1080, 720);
+    });
+
+    function waitForLoad(iteration: number = 0) {
+        cy.wait(1000).then(() => {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['cy' is not defined.](https://app.codacy.com/gh/Project-Lavinia/Lavinia-client/pullRequest?prid=5611900)